### PR TITLE
ODATA-8: Onboard Odata to Papertrail Service for SSDLC Compliance

### PR DESCRIPTION
**Pull request by sanych-sun**
ODATA-16: Generate ssdlc_compliance_report.md and complete policy checklist for releases

### CODE REVIEW

File: evergreen/evergreen.yml [hunk 1]
```diff
@@ -236,10 +251,49 @@ functions:
         working_dir: mongo-aspnetcore-odata
         env:
           NUGET_KEY: ${nuget_key}
+          NUGET_SIGN_CERTIFICATE_FINGERPRINT: ${$NUGET_SIGN_CERTIFICATE_FINGERPRINT}
```
Comment (BorisDog) [inline evergreen/evergreen.yml:254]: is ${$NUGET_SIGN_CERTIFICATE_FINGERPRINT} right?
and not `${NUGET_SIGN_CERTIFICATE_FINGERPRINT}`

**Review by BorisDog — CHANGES_REQUESTED** [review]
Minor comment

File: evergreen/evergreen.yml (hunk 1 above)
Comment (sanych-sun) [reply evergreen/evergreen.yml]: Good catch! Thank you!

File: evergreen/evergreen.yml [hunk 2]
```diff
@@ -235,11 +245,51 @@ functions:
...
+        aws_session_token: ${AWS_SESSION_TOKEN}
+        local_file: ./mongo-aspnetcore-odata/artifacts/ssdlc/ssdlc_compliance_report.md
+        remote_file: mongo-aspnetcore-odata/${PACKAGE_VERSION}/ssdlc_compliance_report.md
+        bucket: csharp-driver-release-assets
+        region: us-west-2
+        permissions: private
+        content_type: text/markdown
+        display_name: ssdlc_compliance_report.md
+
+
```
Comment (BorisDog) [inline evergreen/evergreen.yml:293]: Remove extra line?

**Review by BorisDog — APPROVED** [review]
LGTM + minor comment

**Pull request merged**